ca_gazebo/worlds/apriltags.world

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0"?>
+<sdf version='1.6'>
+  <world name='default'>
+    <include>
+      <uri>model://ground_plane</uri>
+    </include>
+    <include>
+      <uri>model://sun</uri>
+      <pose>0 0 5000 0 0 0</pose>
+      <name>sun</name>
+    </include>
+    <physics type="ode">
+      <real_time_update_rate>1000.0</real_time_update_rate>
+      <max_step_size>0.001</max_step_size>
+      <real_time_factor>1</real_time_factor>
+      <ode>
+        <solver>
+          <type>quick</type>
+          <iters>150</iters>
```

Comment:
Compare this value with other launch files. Use the same approach.

Reply:
This value can be decreased/increased depending on the complexity of the world. But it's fine :+1: